core/mgmt/v1beta/mgmt.proto

```diff
@@ -82,6 +81,38 @@ enum OnboardingStatus {
   ONBOARDING_STATUS_COMPLETED = 2;
 }
 
+// UserProfile describes the public data of a user.
+message UserProfile {
+  // Display name.
+  optional string display_name = 1 [(google.api.field_behavior) = OPTIONAL];
+  // Biography.
+  optional string bio = 2 [(google.api.field_behavior) = OPTIONAL];
+  // Avatar in base64 format.
+  optional string avatar = 3 [(google.api.field_behavior) = OPTIONAL];
```

Comment:
❓ should we use `bytes` instead?
```suggestion
  optional bytes avatar = 3 [(google.api.field_behavior) = OPTIONAL];
```

Reply:
Console can only accept a base64 string. Therefore, we encode it in base64 in the format `data:image/png;base64,xxxxxxx`.